drake/systems/plants/constraint/RigidBodyConstraint.h

```diff
@@ -427,7 +432,9 @@ class DRAKERIGIDBODYCONSTRAINT_EXPORT GazeConstraint : public SingleTimeKinemati
     virtual ~GazeConstraint(void){};
 
 public:
+#ifndef SWIG
```

Comment:
if this is going to appear everywhere, and it's just wrapping a macro, why not make a new macro that includes the swig logic.  (in fact eigen does have EIGEN_MAKE_ALIGNED_OPERATOR_NEW_IF(NeedsToAlign) already defined)